addon/components/sortable-item.js

```diff
@@ -43,12 +42,11 @@ export default class SortableItemComponent extends Component {
     return isEqual(currentOverItem, this);
   }
 
-	@computed
   get documentWindow() {
-    return document.querySelector('body');
+    return document.querySelector(this.args.appendTo || 'body');
```

Comment:
freshservice's ember app is inside body and all styles for ember app are applied only inside this div.
so while creating clone cards to drag instead of body, it has to be inside this ember apps div.